### PR TITLE
let -> const in for...of

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ const letterResult = document.getElementById('letterResult');
 // your browser must support for..of loop
 // and let-scoped variables in for loops
 // const and var could also be used
-for (let letter of eArr) {
+for (const letter of eArr) {
   const li = document.createElement('LI');
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:59]: NIT:
```suggestion
  const li = document.createElement('li');
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 2]
```diff
@@ -260,9 +268,7 @@ new Set("firefox")  // Set(6) { "f", "i", "r", "e", "o", "x" }
 ```js
 const array = Array
   .from(document.querySelectorAll('[id]'))
-  .map(function(e) {
-      return e.id
-  });
+  .map((e) => e.id);
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:271]: The first time we use _implicit no return statement_, for an arrow function. (Just a note)

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Nice. 1 nit and we can land this.

File: files/en-us/web/javascript/reference/global_objects/set/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/set/index.md:271]: Searching for `\(.*\) => [^{]` revealed quite a few others... 😄

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/javascript/reference/global_objects/set/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/set/index.md:271]: Yes, we were already using this. I listed it in our discussion about modern JS.

**Pull request merged**